.github/workflows/build-executable.yml

```diff
@@ -0,0 +1,35 @@
+name: Bulid Executable
+
+on:
+  workflow_call:
+    inputs:
+      release-tag:
+        type: string
+        description: 'Release tag'
+        required: true
+
+jobs:
+
+  standalone-executable:
+    name: Create a standalone linux executable
+    runs-on: ubuntu-22.04
+    if: ${{ inputs.release-tag != ''}}
```

Comment:
Not sure if this is correct: Now you skip the whole job if the release tag is empty. 
Wouldn't it be better to skip only the "Upload to release" step?